restore-dotnet/action.yml

```diff
@@ -9,6 +9,9 @@ inputs:
     description: "Version of dotnet to use. Default is v7.x."
     required: false
     default: "7.x"
```

Comment:
Not for this PR, but can we bump this to 8 now?

Reply:
[TKX-1962](https://trakx.atlassian.net/browse/TKX-1962) is the Epic of .NET 8 migration. It seems everything is done.